src/Pages/ProductPage.tsx

```diff
@@ -56,7 +56,7 @@ export function ProductPage() {
             .then((data) => {
                 setUserData(data.data)
         })  
-    },[]);
+    },[params.productID]);
 
     return (
         <div className="bg-lightGray">
```

Comment:
```suggestion
        <div className="bg-lightGray pt-16">
```

Reply:
The fixed nav bar cuts off part of the product page image and product name. Adding some padding here fixes it.